Emby.Server.Implementations/Data/SqliteItemRepository.cs

```diff
@@ -2723,18 +2710,17 @@ private string FixUnicodeChars(string buffer)
 
         private void AddItem(List<BaseItem> items, BaseItem newItem)
         {
-            var providerIds = newItem.ProviderIds.ToList();
-
             for (var i = 0; i < items.Count; i++)
             {
                 var item = items[i];
 
-                foreach (var providerId in providerIds)
+                foreach (var providerId in newItem.ProviderIds)
```

Comment:
this doesn't feel right, it would loop over `newItem.ProviderIds` for each `item` in `items`. previous approach seemed faster.

Reply:
Wait what? I only removed a copy (ToList)

Reply:
I was under impression that `newItem.ProviderIds` might be some funny iterator which was cached via that `ToList()`. If that is false then this change is okay